docs/ScopeContext/3_public_interfaces.rst

```diff
@@ -10,14 +10,14 @@ Interfaces to other modules will be realized using ROS (rosjava). Currently 2 in
 	# argument: String payload
 	# returns: Bool success
 
-	rosservice call /write_qr
+	rosservice call /roboy/cognition/memory/data/write
```

Comment:
what are the arguments for this service call?

also mention that DataQuery.srv is inside roboy_communication_cognition